src/components/media_manager/src/media_manager_impl.cc

```diff
@@ -213,9 +218,9 @@ void MediaManagerImpl::StartMicrophoneRecording(int32_t application_key,
   std::vector<uint8_t> buf;
   if (file_system::ReadBinaryFile(record_file_source, buf)) {
     if (file_system::Write(file_path, buf)) {
-      LOG4CXX_INFO(logger_,
-                   "File " << record_file_source << " copied to "
-                           << output_file);
+      LOG4CXX_INFO(
+          logger_,
+          "File " << record_file_source << " copied to " << output_file);
```

Comment:
These changes might conflict with the version of `clang-format` that is currently used by the project.